python/ray/autoscaler/autoscaler.py

```diff
@@ -213,7 +219,7 @@ def _update(self):
             num_launches = min(max_allowed, target_workers - num_workers)
             self.launch_new_node(num_launches,
                                  self.config.get("worker_default_node_type"))
-            nodes = self.workers()
+            nodes = self.managed_workers()
```

Comment:
should this be workers() or managed_workers()?

Reply:
i believe this is correct. the semantics are that we ignore the unmanaged node for everything except resource requests.